packaging/pkg_helpers.bash

```diff
@@ -52,6 +52,13 @@ setup_cuda() {
 
```

Comment:
Also needs to change https://github.com/pytorch/builder/blob/master/packaging/README.md